src/sbd-inquisitor.c

```diff
@@ -1101,6 +1116,39 @@ int main(int argc, char **argv, char **envp)
 		goto out;
 	}
 
+	if (reboot_action) {
+		char *p[2];
+		int i;
+		char c;
+		int nrflags = sscanf(reboot_action, "%m[a-z],%m[a-z]%c", &p[0], &p[1], &c);
```

Comment:
FYI this introduces a dependency on POSIX.1-2008 (or glibc >= 2.7). Should be fine ...